### DISCUSSION TITLE
CircleCI can't find the latest release

### DISCUSSION THREAD

**Discussion by grndvl1** [AppDevNext/AndroidChart#159] [General]
Could not find com.github.AppDevNext:AndroidChart:3.1.0.17.

![Screenshot 2023-10-16 at 11 21 36 AM](https://github.com/AppDevNext/AndroidChart/assets/739862/7151996d-6947-4d4f-b3ce-60d70e0f3832)

**Comment by hannesa2** [comment]
On https://jitpack.io/#AppDevNext/AndroidChart there is a build and artifacts for it.
On other libraries I found similar issues. Mostly I updated all to Android Studio Giraffe and the errors are gone
